browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -3891,6 +3877,9 @@ L.CanvasTileLayer = L.Layer.extend({
 
 	// Scrolls the view to selected position
 	scrollToPos: function(pos) {
+		if (pos.pX) // Turn into lat/lng if required.
```

Comment:
please add comment above function that it can take old lat/lng and also new coordinate object as input so it will be obvious